### DISCUSSION TITLE
Hot reload doesn't reload

### DISCUSSION THREAD

**Discussion by newor0599** [flet-dev/flet#1956] [Q&A]
### Question

Hello, i use flet hot reload function by doing `flet run path\to\python\file` , when i save my code with some error with python, flet won't refresh even i fixed the error and saved the file. The error was all printed in the flet cmd (window 11)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Thats a true concern!
#1879
